src/gtest/test_mempool.cpp

```diff
@@ -92,6 +93,8 @@ TEST(Mempool, PriorityStatsDoNotCrash) {
 }
 
 TEST(Mempool, TxInputLimit) {
+    SelectParams(CBaseChainParams::TESTNET);
```

Comment:
Why is this necessary?

Reply:
The tx fails CheckValidation when calling AcceptToMemoryPool, which is the intended behaviour. However, because we now have to pass in a branch id to CheckValidation, Params().GetConsensus() is invoked *before* calling CheckValidation, and Params() hasn't been set up yet, so then we get an error.